dts/bindings/spi/microchip,xec-qmspi-full-duplex.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2018, Google LLC.
+# Copyright (c) 2022, Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Microchip XEC QMSPI controller with local DMA
+
+compatible: "microchip,xec-qmspi-full-duplex"
+
+include: [spi-controller.yaml, pinctrl-device.yaml]
```

Comment:
Consider extracting common properties from `microchip,xec-qmspi-full-duplex` and `microchip,xec-qmspi-ldma` into a shared include

Reply:
will consider in a separate PR later, will like to get this PR merged soon

Reply:
We shouldn't introduce a new file that's almost identical to one that already exists in the tree:
```diff
$ diff microchip,xec-qmspi-ldma.yaml microchip,xec-qmspi-full-duplex.yaml 
2c2
< # Copyright (c) 2021, Microchip Technology Inc.
---
> # Copyright (c) 2022, Microchip Technology Inc.
7c7
< compatible: "microchip,xec-qmspi-ldma"
---
> compatible: "microchip,xec-qmspi-full-duplex"
43c43,48
<         or 4 indicate dual or quad I/O modes. Defaults to 1.
---
>         or 4 indicate dual or quad I/O modes.
>         Defaults to 1 for current driver's support for full-duplex spi.
>       enum:
>         - 1
>         - 2
>         - 4
45c50
<     port_sel:
---
>     port-sel:
54c59
<     chip_select:
---
>     chip-select:
```

Can you go ahead and fix this while the merge window is closed?

Reply:
yes, working on this

Reply:
moved all common properties to a shared include; added as a separate commit

Reply:
The shared include commit needs to be first in the series.

Reply:
there are only 2 commits in the PR, shall I merge the shared include commit with previous commit?

Reply:
also with only the ldma driver (prior to this PR), there is no other driver to compare for moving common properties to a separate include.

Reply:
moved the shared include commit to be first in the series.